cwd/editor.js

```diff
@@ -1,3 +1,60 @@
+const update = (btn) => {
+  const viewNum = btn.id.substring(8, 9);
+  const gaugeNum = btn.id.substring(10, 11);
+  const msgNum = btn.id.substring(12, );
+
+  const prob = [];
+  for (const x of Array(5).keys()) {
```

Comment:
Is this just looping `0, 1, 2, 3, 4`?  It feels like a kind of expensive way of doing that.

Reply:
This does not matter.